### PR TITLE
Update Threat Intel beta with the latest changes

### PR DESCRIPTION
**Pull request by jpipkin1**
## Purpose of this pull request

This pull request updates the following article with the latest changes in the beta:
https://help.sumologic.com/docs/platform-services/threat-intelligence-indicators/

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## PLEASE READ

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [x] Yes, I've run `yarn install`
- [ ] No, does not apply to me

### CODE REVIEW

File: docs/platform-services/threat-intelligence-indicators.md [hunk 1]
```diff
@@ -36,30 +38,44 @@ Add the following capabilities:
...
+:::note
+* The limit of the number of indicators that can be uploaded in one API call is 100.
+* When you add indicators, the event is recorded in the Audit Event Index. See [Audit logging for threat intelligence](#audit-logging-for-threat-intelligence).
+:::
+
+## Typical workflow
+
+Here is the typical workflow to set up and use threat intelligence indicators:
+
+1. A system administrator sets up services to automatically [ingest threat intelligence indicators](#ingest-threat-intelligence-indicators). For example, install a collector such as the [STIX/TAXII 2 Client Source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/stix-taxii-2-client-source), and set up services to obtain indicators from Federal, vendor, and open services and ingest them using the [Threat Intel Ingest Management](https://api.sumologic.com/docs/#tag/threatIntelIngest) APIs. You can manually add additional indicators as needed, such as your own private indicators, using the [Threat Intelligence tab](#add-indicators-in-the-threat-intelligence-tab) or the APIs.
```
Comment (JV0812) [inline docs/platform-services/threat-intelligence-indicators.md:57]:
```suggestion
A system administrator sets up services to automatically [ingest threat intelligence indicators](#ingest-threat-intelligence-indicators). For example, install a collector such as the [STIX/TAXII 2 Client Source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/stix-taxii-2-client-source) and set up services to obtain indicators from Federal, vendor, and open services, later ingest them using the [Threat Intel Ingest Management](https://api.sumologic.com/docs/#tag/threatIntelIngest) APIs. You can manually add more indicators as needed, such as your own private indicators, using the [Threat Intelligence tab](#add-indicators-in-the-threat-intelligence-tab) or the APIs.
```

File: docs/platform-services/threat-intelligence-indicators.md [hunk 2]
```diff
@@ -254,9 +306,25 @@ Since different sources can report different reputations, each source has a repu
 
 <img src={useBaseUrl('img/platform-services/threat-indicators-in-cloud-siem-ui.png')} alt="Threat indicators in the Cloud SIEM UI" style={{border: '1px solid gray'}} width="400" />
 
+### Custom threat intelligence sources in Cloud SIEM
+
+Previously, you could [create custom threat intelligence sources using Cloud SIEM](/docs/cse/administration/create-custom-threat-intel-source/). You can no longer create custom sources using Cloud SIEM, but you must use the Threat Intelligence tab, a collector, or the API to [ingest threat intelligence indicators](#ingest-threat-intelligence-indicators). If you have custom sources in Cloud SIEM, they will continue to be honored until the feature is deprecated at a future time.
```
Comment (JV0812) [inline docs/platform-services/threat-intelligence-indicators.md:311]: Do we talk about previously and currently in para? If so, can we mention **Currently or now** for the second line. Just to show the difference.

**Review by JV0812 — APPROVED** [review]
Minor fixes. I will approve the PR @jpipkin1, check my comments and if it is relevant please fix those.

File: docs/platform-services/threat-intelligence-indicators.md (hunk 2 above)
Comment (jpipkin1) [reply docs/platform-services/threat-intelligence-indicators.md]: Good point. I'll update the wording.

**Pull request merged**